components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -233,23 +207,23 @@ public void deleteBusinessOwner(String businessOwnerId) throws AppManagementExce
             }
             connection = APIMgtDBUtil.getConnection();
 
-            String queryToDeleteRecordTwo = "DELETE FROM BUSINESS_OWNER_CUSTOM_PROPERTIES WHERE OWNER_ID = ?";
+            String queryToDeleteRecordTwo = "DELETE FROM APM_BUSINESS_OWNER_CUSTOM_PROPERTIES WHERE OWNER_ID = ?";
 
             statementToDeleteRecordTwo = connection.prepareStatement(queryToDeleteRecordTwo);
             statementToDeleteRecordTwo.setString(1, businessOwnerId);
             statementToDeleteRecordTwo.executeUpdate();
 
-            String queryToDeleteRecord = "DELETE FROM BUSINESS_OWNER WHERE OWNER_ID = ?";
+            String queryToDeleteRecord = "DELETE FROM APM_BUSINESS_OWNER WHERE OWNER_ID = ?";
 
             statementToDeleteRecord = connection.prepareStatement(queryToDeleteRecord);
             statementToDeleteRecord.setString(1, businessOwnerId);
-            statementToDeleteRecord.executeUpdate();
+            int i = statementToDeleteRecord.executeUpdate();
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/commit/9b1e766e9063423faaf1b86f353a0db0d54deb12